talks/_posts/2017-11-10-intro-to-haskell.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Freedom From State: An Intro to Haskell
+author: Josh Bicking
+layout: post
+slides: /talks/2017-w11-intro-to-haskell.pdf
```

Comment:
A small typo here. The slides are named `/talks/2017-w11-fall-intro-to-haskell.pdf`.